port.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+#
+# Simple shell script to invoke MacPorts from the source directory
+#
+
+BASEDIR=$(cd $(dirname "$0"); pwd)
```

Comment:
Need more quotes to properly handle paths that might contain spaces.
```suggestion
BASEDIR=$(cd "$(dirname "$0")"; pwd)
```

Reply:
Simplied by using `autoconf`; fixed.